### PR TITLE
support rm agreed cal

### PR DESCRIPTION
**Pull request by wwxFromTju**

### CODE REVIEW

File: eval/analyze_rm.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+datasets.disable_progress_bar()
+
+
+def check_rm_agree(directory_path):
+    csv_files = [
+        file_name
+        for file_name in glob.glob(directory_path + "/**/*.csv", recursive=True)
+        if "gpt-3.5-turbo-0125" in file_name
+    ]
+    datas = datasets.load_dataset("csv", data_files=csv_files)["train"]
```
Comment (vwxyzjn) [inline eval/analyze_rm.py:11]: WDYT if we calculate the agreement rate for each random seed and produce an error bar similar to the following. 


image

**Review by vwxyzjn — COMMENTED** [review]
Nice work! A minor comment.

File: eval/analyze_rm.py (hunk 1 above)
Comment (wwxFromTju) [reply eval/analyze_rm.py:11]: Get, I will handle this request

Comment (wwxFromTju) [reply eval/analyze_rm.py:11]: support：6c15ef87011f03cbe1f26c3a64610e2f75219811

**Review by vwxyzjn — APPROVED** [review]
LGTM! Thanks @wwxFromTju !!!

**Pull request merged**